### PR TITLE
Credit Button Demo

### PR DESCRIPTION
**Pull request by vishakha94**

### CODE REVIEW

File: demo/app/client/js/patterns/credit.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                client: {
+                    sandbox:    'AZDxjDScFpQtjWTOUtWKbyN_bDt4OgqaF4eYXlewfBP4-8aqX3PiV8e1GWU6liB2CUXlkA59kJXE7M6R',
+                    production: 'Aco85QiB9jk8Q3GdsidqKVCXuPAAVbnqm0agscHCL2-K2Lu2L6MxDU2AwTZa-ALMn_N0z-s2MXKJBxqJ'
+                },
+
+                // Wait for the PayPal button to be clicked
+
+                payment: function() {
+
+                return paypal.rest.payment.create(this.props.env, this.props.client, {
```
Comment (bluepnume) [inline demo/app/client/js/patterns/credit.jsx]: Could we fix the indentation here?

File: demo/app/client/js/patterns/styles.jsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ export let styles = {
     description: (
         <div>
             <p>First, a button is created using <span className="pre">paypal.Button.render()</span>, and rendered to the <span className="pre">#paypal-button-container</span> element.</p>
-            <p>Along with the other required options, we pass a <span className="pre">style</span> block with various options to customize the button. In this block I can specify the <span className="pre">size</span>, <span className="pre">color</span> and <span className="pre">shape</span> of the button</p>
```
Comment (bluepnume) [inline demo/app/client/js/patterns/styles.jsx]: Can we also list `style: 'checkout' // checkout | credit` to the example code for this section?

File: demo/app/client/js/patterns/styles.jsx [hunk 3]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 
 export let styles = {
 
-    name: `Button Styles`,
+    name: `Checkout Button`,
```
Comment (bluepnume) [inline demo/app/client/js/patterns/styles.jsx]: Let's keep this as Button Styles, since it's about more than just the label.

**Comment by bluepnume** [comment]
Looks good! Just a couple of comments then will merge :)

File: demo/app/client/js/patterns/styles.jsx (hunk 2 above)
Comment (vishakha94) [inline demo/app/client/js/patterns/styles.jsx]: Done

File: demo/app/client/js/patterns/credit.jsx (hunk 1 above)
Comment (vishakha94) [inline demo/app/client/js/patterns/credit.jsx]: DOne

**Pull request merged**